composer.json

```diff
@@ -4,7 +4,14 @@
     "type": "library",
     "license": "MIT",
     "config": {
-        "sort-packages": true
+        "platform": {
+            "php": "7.4.7"
+        },
```

Comment:
This shouldn't be here

Reply:
Ah yes, remainder of my multiple-php-versions struggles, I'll remove it